i18n/en.json

```diff
@@ -156,6 +156,13 @@
 	"smw-property-name-invalid": "Property $1 can not be used (invalid property name).",
 	"smw-sp-property-searchform": "Display properties that contain:",
 	"smw-sp-property-searchform-inputinfo": "The input is case sensitive and when used for filtering, only properties that match the condition are displayed.",
+	"smw-special-property-searchform": "Display properties that contain:",
+	"smw-special-property-searchform-inputinfo": "The input is case sensitive and when used for filtering, only properties that match the condition are displayed.",
+	"smw-special-property-searchform-options": "Options",
+	"smw-special-wantedproperties-filter-label": "Filter:",
+	"smw-special-wantedproperties-filter-none": "None",
+	"smw-special-wantedproperties-filter-unapproved": "Unapproved",
+	"smw-special-wantedproperties-filter-unapproved-desc": "Filter option used in connection with the authority mode.",
```

Comment:
@mwjames Hmm, where does this message show up? Cannot find in on special "WantedProperties"